migrations/staging/20171017104000_stg_court_reporters.js

```diff
@@ -0,0 +1,28 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.withSchema('staging').createTable('court_reporters', function (table) {
```

Comment:
If you merge in `wmt-175/merge` this should disappear

Reply:
merged